students/liikt/problems.csv

```diff
@@ -0,0 +1,12 @@
+5+5,10
+1+1,2
+8+3,11
+1+2,3
+8+6,14
+3+1,4
+1+4,5
+5+1,6
+2+3,5
+3+3,6
+2+4,6
+5+2,7
```

Comment:
Git likes an empty new line at the end of all files, let's add it there.

Reply:
Yeah i was thinking about adding one. I just may have forgotten it